package.json

```diff
@@ -26,14 +30,21 @@
   },
   "homepage": "https://github.com/MINERVA-MD/minerva-core#readme",
   "devDependencies": {
-    "@electron-forge/cli": "^6.1.1",
```

Comment:
Had a build issue w/ `electron-forge` so opted to swap it out for `electron-builder`,  just temporarily. Can re-assess what builder we want - quite easy to swap out.

Reply:
I've heard good things about electron-builder's optimization so I'm cool with it 👍